components/blocks/FilesToCards.vue

```diff
@@ -284,6 +279,23 @@ export default {
     clearAll() {
       this.searchGroup = [];
     },
+    contributors(card) {
+      const subset = (({investigators, people}) => ({investigators, people}))(card)
+      return subset
+    },
+    contributorLink(contributor) {
+      if ('link' in contributor) {
+        return contributor.link
+      } else if ('github_user' in contributor) {
+        return 'https://github.com/' + contributor.github_user
+      } else {
+        return undefined
+      }
+    },
+    contributorIcon(contributorType) {
```

Comment:
instead of having this as a function, make `contibutorIcon` data with the object on 296, then can just do `contributerIcon[contributorType]`